modules/ROOT/pages/import-export-packages.adoc

```diff
@@ -56,18 +56,34 @@ image::import-export-packages-a1b21.png[]
 |*Anypoint Studio Project to Mule Deployable Archive (includes Studio metadata)* |Exports a Mule project to a `.jar` deployable file.
 |===
 
-=== Exporting a Studio Project to a Mule Deployable File
+=== Exporting a Studio Project to a Deployable Mule Application
 
 . Select *Export* from the *File* menu.
 . In the Import wizard, click to expand the Mule folder, and select *Anypoint Studio Project to Mule Deployable Archive (includes Studio metadata)*, then click Next.
 . Select the project that you want to export, and click *Next*.
 . In the *JAR file* drop-down menu, click the ellipses (...) button to explore your local drive, and select the folder to which you want to export your JAR deployable file. +
-Additionally, you can choose *Attach Project Sources* to include metadata that Studio requires to re-import the deployable file as an open Mule project into your workspace. +
-This is the same as using the `-DattachMuleSources` flag when packaging using the Mule Maven Plugin. See xref:mule-runtime::mmp-concept.adoc#mule-application-packaging[Mule Application Packaging] for more information.
+Additionally, you can choose *Attach Project Sources* to include metadata that Studio requires to re-import the deployable file as an open Mule project into your workspace.
 +
-You can also choose *Include Project Modules and Dependencies*. +
-This option creates a heavy package with all the dependencies of your project already packaged in your app. This is required to deploy your app to CloudHub. +
-If you uncheck this option, Studio exports your project in a lightweight package. This is useful if you want to distribute your project to other Studio installations because it only keeps a reference to all its dependencies. When re-importing it to Studio, all your dependencies will be automatically downloaded.
+[NOTE]
```

Comment:
See style reference re. use of notation formats like "Note." Always evaluate whether this should be part of the actual narrative flow, because some people skip notes and others are distracted by them breaking up narrative flow.